test/runner.test.js

```diff
@@ -1,38 +1,102 @@
 const fs = require("fs");
 const path = require("path");
 const execa = require("execa");
+const dircompare = require("dir-compare");
 
+const PACKAGES_DIR = path.resolve(__dirname, "..", "packages");
 const TEMPLATES_DIR = path.resolve(__dirname, "..", "templates");
 
+const packages = fs.readdirSync(PACKAGES_DIR);
 const templates = fs.readdirSync(TEMPLATES_DIR);
 
+const pkgNames = [];
+
 describe("npx create-snowpack-app", () => {
+  beforeAll(async () =>
+    // use yarn link to use local packages on installs
+    Promise.all(
+      packages.map(async (dir) => {
+        // read package name
+        const cwd = path.join(PACKAGES_DIR, dir);
+        const pkgJSON = JSON.parse(
+          fs.readFileSync(path.join(cwd, "package.json")),
+          "utf8"
+        );
+        pkgNames.push(pkgJSON.name);
+
+        // run yarn link
+        await execa("yarn", ["link"], { cwd });
+      })
+    ).then(() =>
+      // iterate through templates and make sure node_modules are installed
+      Promise.all(
+        templates.map(async (template) => {
+          const cwd = path.join(TEMPLATES_DIR, template);
+
+          // link all local packages
+          await execa("yarn", ["link", ...pkgNames], { cwd });
+
+          // run yarn in this directory
+          await execa("yarn", { cwd });
+        })
+      )
+    )
+  );
```

Comment:
This is a little annoying to have to `yarn link` everything, and `yarn` install everything, but it does give us the added assurance that we’re testing local changes and not running a test suite against npm (which would be frustrating, because if an npm build was failing, all our PRs would be, too 😅 )

Reply:
Doesn't running `yarn` in the top-level project do all of this for you? Running `yarn link` ourselves makes me nervous...

Reply:
Oh I think you’re right. I started doing that because I (wrongly) thought we had to run `yarn` in every directory (caused by an error I encountered that turned out to not be dependency-related).

However, if we just run `yarn` in the top-level, you’re right that Lerna figures out how to symlink everything locally. That’s cleaner!

Reply:
woo!